LinkedIn-Certifications-Manager/README.md

```diff
@@ -0,0 +1,20 @@
+# LinkedIn Certification Manager
+
+A CLI based tool to automatically add certification to your LinkedIn profile. 
+
+## How to use
+- Make sure you have Google Chrome browser.
+- Download [chromedriver](https://chromedriver.chromium.org/)  for your version of Chrome and move it to LinkedIn-Certification-Manager directory  
+- Add your linkedin credentials in `credentials.json`.
+- Add course data in `data.json`.
```

Comment:
You need to take this data as input from the user. So you can delete the corresponding files and this two instructions too